clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_utils.py

```diff
@@ -0,0 +1,228 @@
+# coding: utf8
+
+"""Deeplearning prepare data - Clinica Utilities.
+This file has been generated automatically by the `clinica generate template`
+command line tool. See here for more details:
+http://clinica.run/doc/InteractingWithClinica/
+"""
+
+
+def extract_slices(preprocessed_T1, slice_direction=0, slice_mode='original'):
+    """
+    This function extracts the slices from three directions
+    :param preprocessed_T1:
+    :param slice_direction: which axis direction that the slices were extracted
+    :return:
+    """
+    import torch
+    import os
+
+    image_tensor = torch.load(preprocessed_T1)
+    # reshape the tensor, delete the first dimension for slice-level
+    image_tensor = image_tensor.view(image_tensor.shape[1], image_tensor.shape[2], image_tensor.shape[3])
+
+    # sagital
+    slice_list_sag = range(20, image_tensor.shape[0] - 20)  # delete the first 20 slice and last 20 slices
+
```

Comment:
I am aware that this corresponds to not relevant slices (i.e. does not contain brain voxels) but it is likely that the `extract_slices` function can generalise to other types of inputs 🤔 

I am wondering if this should be optional parameters of this function where we could remove the first `N` slices and/or the last `M` slices.

What do you think @mdiazmel?

Reply:
Yes, it's absolutely possible to add this as an optional parameter of the function.  But I think that we need the advice of someone expert in this kind of images (@nburgos, maybe) to determine the possible ranges for`N` and `M`...

Reply:
We will discuss about it this afternoon in that case :)

Reply:
For the final version, all slices are extracted. Slice selection is done in the training.